pages/index.tsx

```diff
@@ -1,14 +1,15 @@
-import { signIn, useSession } from "next-auth/react"
+import { useSession } from "next-auth/react"
 
 function HomePage() {
   const { data: session } = useSession()
 
-  return <div>
-    Welcome to SalesWell{session ? ' ' + session.user.name : ''}!
-    <br />
-    { !session ? 
-      <button onClick={() => signIn()}>Sign in with Email</button> 
-      : null }
+  return <div style={{
```

Comment:
if we want css let's use a css variable or even a css file. Also can't we do this dirrectly by using MaterialUi features?